content/help/ops/perf/index.md

```diff
@@ -0,0 +1,94 @@
+sidecars' scope includes the whole mesh. Enabling namespace isolation significantly
+reduces sidecar memory usage and Pilot's CPU usage.
+
+## Istio default configuration values vs. functionality
+
+To provide good performance out of the box, the following features are disabled by default in Istio 1.1:
+
+| Feature | Default | Description |
+| --- | --- | --- |
+| `global.proxy.accessLogFile` | `""` |  Since proxy access logging contributes to high CPU usage and I/O traffic, logging is disabled by default. To enable proxy access logging, set this value to `/dev/stdout`. |
+| `mixer.adapters.stdio.enabled` | `false` |  Since Mixer access logging contributes to high CPU usage and I/O traffic, this debugging feature is disabled by default. To enable access Mixer access logging, set this value to `true`. |
+| `global.disablePolicyChecks` | `true` | To avoid unnecessary latency in the data path, policy checks are disabled by default. When you add traffic management policies, set this value to `true` to ensure Pilot performs the checks. |
+
+## Namespace isolation
+
+Namespace isolation provides a major performance improvement in Istio 1.1. To implement namespace isolation, use a [sidecar configuration object](/docs/reference/config/networking/v1alpha3/sidecar).
+Enabling namespace isolation has the following consequences for your mesh:
+
+- The default connectivity is namespace-wide instead of mesh-wide.
+- Pilot configures Envoy with only the relevant information to route traffic within the namespace.
+- The Envoy proxy receives less configuration information lowering the proxy's memory usage.
+
+The following table document shows the configuration parameters of two different meshes: Setup 1 and Setup 2.
+
+
+| Parameter | Setup 1 | Setup 2 |
+| --- | --- | --- |
+| Namespaces (N)| 40 | 50 |
+| Sidecars (N x 40) | 1600 | 2000 |
+| Services (N x 20) | 800 | 1000 |
+| Virtual Services (N x 2) | 80 | 100|
+| Gateways (N) | 40 | 50 |
+| Configuration change rate (N/4 per minute) | 10/min | 12/min |
+
+The following table shows the impact enabling namespace isolation has in the resource consumption of Setup 1 and Setup 2:
+
+| Resource | Setup 1 || Setup 2 ||
+| --- | --- | --- | --- | --- |
+| Namespace Isolation | Enabled |  Disabled | Enabled | Disabled |
+| Pilot replicas | 1  | 2 | 1 | 3 |
+| Pilot CPU      | 1 vCPU | 5 vCPU | 1 vCPU | 8 vCPU |
+| Pilot Memory   | 1.5 GB | 4 GB | 1.5 GB | 8 GB |
+| Proxy Memory (per instance) | 50 MB  | 170 MB | 50 MB  | 225 MB |
+
+## Size and capacity planning
+
+Using the available the performance data, you can make educated sizing and capacity planning decisions for your Istio mesh.
+Your capacity planning effort depends on the following factors among many others:
+
+* The kind of application running in the mesh.
+* The transaction volume of the mesh.
+* The Istio features enabled. 
+
+You need to consider capacity from many angles: 
```

Comment:
`Capacity should be considered from many vectors: `might be better.  Angle is an idiom - vector is not.